embedded-workshop-book/src/listing-usb-devices.md

```diff
@@ -1,13 +1,13 @@
 # Listing USB Devices
 
-✅ To list all USB devices, run `usb-list`.
+✅ To list all USB devices, run `cargo xtask usb-list` from the `advanced` folder.
```

Comment:
do we want to add a little 🔎 section that explains what this means plus a link to https://github.com/matklad/cargo-xtask in 1-2 sentences?